src/index.js

```diff
@@ -280,7 +281,7 @@ export default class GoCommerce {
       }));
     } else {
       return Promise.reject(
-        "Invalid paymentDetails - must have an order_id, an amount and a stripe_token or a paypal_payment_id and paypal_user_id"
+        "Invalid paymentDetails - must have an order_id, an amount, a provider, and a stripe_token or a paypal_payment_id and paypal_user_id"
```

Comment:
It depends on how much we want to do in the client. It is possible to attempt to use PayPal, but not have it enabled on the server side.